libraries/ms-drivers/inc/drv120_relay.h

```diff
@@ -1,17 +1,28 @@
 #pragma once
 
 // Driver for the DRV120 relay
-// Requires GPIO to be initialized
+// Requires GPIO and Gpio_it to be initialized
 
 #include <stdbool.h>
+#include <stdlib.h>
 
 #include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
 #include "status.h"
 
+typedef struct Drv120RelaySettings {
+  GpioAddress *pin;        // Pin initialized as output for relay
+  GpioAddress *status;     // Pin set to trigger interrupt on error
+  GpioItCallback handler;  // Error handler
+  void *context;           // Context for error handler
```

Comment:
Nitpicks on the names:
* I know `pin` was the parameter name for `drv120_relay_init`, but it's vague now since there's more than one pin. I'd prefer `enable_pin` or something since it's the relay enable pin.
* Similarly, could we call `status` instead `status_pin`? Seeing `.status = ...` in a settings struct initialization would make me think it takes a StatusCode.
* If we call `handler` instead `error_handler` or `error_callback` then you don't need the comment on it anymore.